### DISCUSSION TITLE
Managed Identity Support for Storage Connection

### DISCUSSION THREAD

**Discussion by haaaow** [microsoft/DurableFunctionsMonitor#189] [Q&A] [answered]
Hi, I'm exploring to host one standalone DFM to connect with multiple task hubs in different storage accounts. Looks like it is supported with DFM_ALTERNATIVE_CONNECTION_STRING_my_other_conn_string configs. Does this support Managed Identity authentication to storage account, instead of connection strings?

**Comment by scale-tone** [comment, marked as answer]
Hi @haaaow , DFM_ALTERNATIVE_CONNECTION_STRING_my_other_conn_string cannot be identity-based - this is not supported.
I can only suggest running multiple DfMon instances, one per each storage account. You can always deploy them into the same App Service Plan, if you're concerned about extra costs.
Or else [DfMon-as-a-VsCode-extension](https://marketplace.visualstudio.com/items?itemName=DurableFunctionsMonitor.durablefunctionsmonitor) is always your friend :)
